ios/ScannerBle.swift

```diff
@@ -25,7 +25,7 @@ import CoreBluetooth
 ///
 class BeaconScanner: NSObject, Scanner, CBCentralManagerDelegate {
   private var centralManager: CBCentralManager!
-  private let beaconOperationsQueue: dispatch_queue_t = dispatch_queue_create("beacon_operations_queue", nil)
+  private let beaconOperationsQueue: dispatch_queue_t = dispatch_queue_create("beacon_operations_queue", DISPATCH_QUEUE_SERIAL)
```

Comment:
Might be good to explain this with a comment as it caused you so much confusion.